chapters/hub/publish-your-pod-image.md

```diff
@@ -59,7 +59,7 @@ Please refer to [hub/examples/mwu_encoder/manifest.yml](hub/examples/mwu_encoder
 
 ## Steps to Publish Your Image
```

Comment:
Change to `## Publish your image`

@FionnD Sentence case is used for headings, right?

Reply:
Yes